app/models/submission.rb

```diff
@@ -14,6 +14,8 @@ class Submission < ApplicationRecord
 
   enum state: [:inactive, :active]
 
+  scope :active, -> { where(state: :active) }
```

Comment:
I think you get this scope for free with the `enum`, same as with the `Activity.project` one..?

Reply:
One day I will learn all of Rails' secrets